flask/app/participants.py

```diff
@@ -26,6 +26,20 @@
 @app.route("/api/participants", methods=["GET"])
 @login_required
 def list_participants():
+
+    # parsing query parameters
+    max_rows = request.args.get("max_rows", default=100, type=int)
```

Comment:
It looks like the behaviour of using type=int here means that if max_rows is passed a string, it instead uses the default value for max_rows rather than passing an error. Not sure if that's a problem or not?

Reply:
ooh good point like you said it still works even if the query is something like max_rows='DFSDFDSF', should I hold off on this until we figure out the input validation mentioned in the retrospective?

Reply:
As long as this and #195 roughly match up it should be okay